uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/contract-entitlements.component.spec.js

```diff
@@ -115,7 +115,7 @@ define([
     function getAbsenceTypeMocks () {
```

Comment:
The function `getAbsenceTypeMocks ()` is dublicated in files 
`js/angular/test/absence-tab/components/contract-entitlements.component.spec.js` and 
`js/angular/test/absence-tab/components/annual-entitlements.component.spec.js`

Can we move it into a common place?